cmd/minishift/cmd/openshift/service_list.go

```diff
@@ -53,6 +74,6 @@ var serviceListCmd = &cobra.Command{
 }
 
 func init() {
-	serviceListCmd.Flags().StringVarP(&serviceListNamespace, "namespace", "n", "default", "The namespace of the services.")
+	serviceListCmd.Flags().StringVarP(&serviceListNamespace, "namespace", "n", "", "The namespace of the services.")
```

Comment:
@hferentschik might have some opinion keeping `default` as default namespace as it was added by him when I kept it empty in my last PR.